src/Dropdown/Option.jsx

```diff
@@ -47,13 +53,21 @@ const DropdownItem = ({
     </Flex>
   )
 
+  useLayoutEffect(() => {
+    if (hasKeyboardFocus) {
+      optionRef.current.scrollIntoView({ block: 'nearest', inline: 'nearest' })
```

Comment:
It looks like we have partial support for `scrollIntoView` on safari, please test this
https://caniuse.com/?search=scrollIntoView

An alternative would be to calculate and set the list container scroll position. I think we can have everything we need (list client height, list scroll height, option height)

Reply:
The partial support refers to `smooth` scroll behaviour. We don't use it so it's not going to be a problem.

Reply:
The partial support only refers to smooth scrolling option, we're in the green here.